### ISSUE TITLE
Incorrect result of transformation inference when I train a custom dataset model

### ISSUE THREAD

**Issue by JunGenius** [QIANXUNZDL123/tensorrtx-yolov7#3]
hello, author
When I retrain, changing the number of classes, the result of inference is incorrect. But using official model inference, the result is correct. Can you help me, how to solve this problem?
Thanks.

**Comment by QIANXUNZDL123** [comment]
> hello, author When I retrain, changing the number of classes, the result of inference is incorrect. But using official model inference, the result is correct. Can you help me, how to solve this problem? Thanks.

maybe you should reparameterization   for exmple   if you train yolov7.pt with your datasets , you need  reparameterization  yolov7.pt , you can see https://github.com/WongKinYiu/yolov7

**Comment by JunGenius** [comment]
> 

Yeah, after I reparameterization, the results are normal. thanks.
